packages/components/psammead-episode-list/src/episode.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import { C_METAL, C_POSTBOX } from '@bbc/psammead-styles/colours';
+
+import { string, shape, func, arrayOf, element } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+
+import PlayButton from './playButton';
+
+const getEpisodeWrapper = ElementType => styled(ElementType)`
+  a {
```

Comment:
Could these styles not be applied to the link?

Reply:
👍 fixed by addition of `EpisodeList.Link` component